components/BottomAppBar/examples/BottomAppBarNavBarExample.m

```diff
@@ -0,0 +1,158 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "BottomAppBarExampleTableViewController.h"
+#import "MaterialBottomAppBar.h"
+#import "MaterialNavigationBar.h"
+#import "MDCButtonColorThemer.h"
+#import "MDCNavigationBarColorThemer.h"
+
+@interface BottomAppBarNavBarViewController : UIViewController
+
+@property(nonatomic, strong) MDCNavigationBar *navBar;
+
+@end
+
+@implementation BottomAppBarNavBarViewController
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  [self setupNavBar];
+}
+
+- (void)setupNavBar {
+  self.navBar = [[MDCNavigationBar alloc] initWithFrame:self.view.bounds];
+  UINavigationItem *navItem = [[UINavigationItem alloc] initWithTitle:@""];
+  [self.view addSubview:self.navBar];
+
+  // Configure the navigation buttons to be shown on the bottom app bar.
+  UIBarButtonItem *barButtonLeadingItem =
+      [[UIBarButtonItem alloc] initWithTitle:@"Back"
+                                       style:UIBarButtonItemStylePlain
+                                      target:self
+                                      action:@selector(didTapLeft:)];
+
+  UIBarButtonItem *barButtonTrailingItem =
+      [[UIBarButtonItem alloc] initWithTitle:@"Right"
+                                       style:UIBarButtonItemStylePlain
+                                      target:self
+                                      action:@selector(didTapRight:)];
+
+  navItem.leftBarButtonItems = @[ barButtonLeadingItem ];
+  navItem.rightBarButtonItems = @[ barButtonTrailingItem ];
+
+  [self.navBar setLeftBarButtonItems:navItem.leftBarButtonItems];
+  [self.navBar setRightBarButtonItems:navItem.rightBarButtonItems];
+
+  MDCBasicColorScheme *clearScheme =
+      [[MDCBasicColorScheme alloc] initWithPrimaryColor:[UIColor clearColor]];
+  [MDCNavigationBarColorThemer applyColorScheme:clearScheme toNavigationBar:self.navBar];
+  self.navBar.tintColor = [UIColor blackColor];
+}
+
+- (void)viewWillLayoutSubviews {
+  [super viewWillLayoutSubviews];
+  self.navBar.frame = self.view.bounds;
```

Comment:
I think you should just use autolayout. This is a little exotic for example code.

Reply:
Done.

Reply:
This looks simple enough to use autoresizing masks.